models/base/src_base.yml

```diff
@@ -15,11 +15,15 @@ sources:
           - name: session_id
             description: '{{ doc("col_session_id") }}'
             tests:
-              - not_null
+              - not_null:
+                  config:
```

Comment:
Yeah I think we could remove them and either check in the `snowplow_mobile_base_events_this_run` table or not check at all. Since we already test in that table I'll remove the tests in the source table